pyproject.toml

```diff
@@ -147,9 +147,9 @@ dev = [
     "numpy<1.24.4; python_version < '3.9'",
     "pytest-recording>=0.13.2",
     "uvicorn>=0.30.6",
-    # Logfire API
     "logfire-api",
     "requests",
+    "setuptools>=75.3.0",
```

Comment:
It's used in this code in `utils.py`:

```python
def get_version(version: str) -> Version:
    """Return a packaging.version.Version object from a version string.

    We check if `packaging` is available, falling back to `setuptools._vendor.packaging` if it's not.
    """
    try:
        from packaging.version import Version

    except ImportError:  # pragma: no cover
        # Trigger the sys.path change mentioned below, but discard this.
        from setuptools._vendor.packaging.version import Version
```

and it's no longer a dependency of OTEL.

We're essentially relying on either `packaging` or `setuptools` being installed. Currently we're safe because `packaging` is a dependency of `opentelemetry-instrumentation`, but there isn't actually much reason for `opentelemetry-instrumentation` to be a required dependency.

Reply:
Anyway, regardless of `packaging` being installed/required, we still need that logic because of the sys.path changes made by setuptools, so we need it as a dev dependency.